interop.py

```diff
@@ -171,6 +171,16 @@ def _run_testcase(self, server: str, client: str, test: Callable[[], testcases.T
       "docker cp \"$(docker-compose --log-level ERROR ps -q sim)\":/logs/. " + sim_log_dir.name,
       shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT
     )
+    # copy logs from the client
+    subprocess.run(
+      "docker cp \"$(docker-compose --log-level ERROR ps -q client)\":/logs/. " + client_log_dir.name,
```

Comment:
presumably this copies all files... I would have expected this to be ":/logs/*" instead of ":/logs/."

Reply:
If I remember correctly, this was some quirk of `docker cp`. Not sure though, I figured that out when making this change for copying the simulator logs, and now just copied it from there.